### DISCUSSION TITLE
Can we avoid showing in console about Powered by Enable3D?

### DISCUSSION THREAD

**Discussion by Smithangshu** [enable3d/enable3d#350] [Help / Questions]
Can we avoid showing in console about Powered by Enable3D?

**Comment by yandeu** [comment]
I believe I added the option to remove this somewhere. But not sure.

Update:

No, sorry. It is hard coded in https://github.com/enable3d/enable3d/blob/master/packages/threeGraphics/src/index.ts :/

**Reply by Smithangshu** [reply]
If I would remove it. Will it violate the license of the library?

**Reply by Smithangshu** [reply]
Or could there be any workaround for it?

**Reply by yandeu** [reply]
No. It would not violate the license. Feel free to modify the source to add a removal option.

**Reply by Smithangshu** [reply]
If you would not mind, can you please add an option to toggle the message. Because I see the source is written in typescript, I have no experience and environment for building typescript source to usable script.

**Reply by Aaryan-Umar** [reply]
Replace this file : https://github.com/enable3d/enable3d/blob/master/packages/threeGraphics/src/index.ts

with this to disable the message

export { ThreeGraphics } from './core'
export { PhysicsLoader } from '@enable3d/common/dist/physicsLoader'
export * as Types from '@enable3d/common/dist/types'

import { VERSION } from './version'
/*
const info = `Powered by enable3d v${VERSION}`
console.log(
  `%c %c %c %c %c ${info} %c https://enable3d.io/`,
  'background: #ff0000',
  'background: #ffff00',
  'background: #00ff00',
  'background: #00ffff',
  'color: #fff; background: #000000;',
  'background: none'
)

*/

**Reply by yandeu** [reply]
Can someone make a PR where you could set 
`new Project({ hidePoweredBy: true })`?
I will be happy to merge.
